src/services/ls-migration/addressBook.ts

```diff
@@ -12,6 +13,9 @@ export const migrateAddressBook = (lsData: LOCAL_STORAGE_DATA): AddressBookState
     console.log('Migrating address book')
 
     const newAb = legacyAb.reduce<AddressBookState>((acc, { address, name, chainId }) => {
+      if (!name || !utils.isAddress(address)) {
```

Comment:
I would add `!name || !address || ` in the beginning because isAddress seems to accept only strings.